app.js

```diff
@@ -0,0 +1,12 @@
+const express = require("express")
+const getTopics = require("./controllers/controller_core")
+
+const app = express()
+
+app.use(express.json())
```

Comment:
Hi Simone, we wouldn't need this line of code yet. Please ensure the code you're submitting is only relevant to the ticket you're currently submitting. Please remove this until it's required.